platform-tools-sdk/cargo-build-sbf/src/main.rs

```diff
@@ -1241,6 +1252,7 @@ fn main() {
         workspace: matches.is_present("workspace"),
         jobs: matches.value_of_t("jobs").ok(),
         arch: matches.value_of("arch").unwrap(),
+        optimize_size: matches.is_present("optimize-size"),
```

Comment:
nit: if you go with the other suggestion
```suggestion
        optimize_size: matches.is_present("optimize_size"),
```